src/inverted_index.h

```diff
@@ -33,6 +33,7 @@ typedef struct InvertedIndex {
   t_docId lastId;
   uint32_t numDocs;
   uint32_t gcMarker;
+  t_fieldMask fieldMask;
```

Comment:
Maybe we can only allocated it if `Index_StoreFieldFlags` is set?

Reply:
Great idea